pkg/apis/agones/v1/common.go

```diff
@@ -28,7 +28,7 @@ import (
 // Block of const Error messages
 const (
 	ErrContainerRequired        = "Container is required when using multiple containers in the pod template"
-	ErrHostPortDynamic          = "HostPort cannot be specified with a Dynamic PortPolicy"
+	ErrHostPortDynamic          = "HostPort cannot be specified with a Dynamic or Passthrough PortPolicy"
```

Comment:
There is a check for both policies so I decided to update this description and cover it in tests
https://github.com/googleforgames/agones/pull/1485/files#diff-5d7500992f221aa57cd7cee64628157bR446

Reply:
That's a good call :+1: wondering if we should rename this variable. Maybe `ErrHostPort` ? WDYT?

Reply:
Yes, it makes sense